test/unit/oracle/CollateralizationOracleGuardian.test.ts

```diff
@@ -0,0 +1,183 @@
+import { getCore, getAddresses, expectRevert, increaseTime, getImpersonatedSigner } from '../../helpers';
+import { expect } from 'chai';
+import hre, { ethers } from 'hardhat';
+import { Signer } from 'ethers';
+import { CollateralizationOracleWrapper, MockCollateralizationOracle } from '@custom-types/contracts';
+import { CollateralizationOracleGuardian } from '@custom-types/contracts/CollateralizationOracleGuardian';
+
+describe.only('CollateralizationOracleGuardian', function () {
+  let userAddress: string;
+  let guardianAddress: string;
+  let governorAddress: string;
+  let oracleWrapper: CollateralizationOracleWrapper;
+  let core: any;
```

Comment:
Theres some issues with this because the helpers file doesn't use types. If you can make a PR to convert that we can start typing Core going forward. While you're at it try to remove dependencies on web3